module/VuFind/src/VuFind/Config/PathResolverFactory.php

```diff
@@ -60,6 +62,25 @@ class PathResolverFactory implements FactoryInterface
      */
     protected $defaultLocalConfigSubdir = PathResolver::DEFAULT_CONFIG_SUBDIR;
 
+    /**
+     * INI file reader
+     *
+     * @var IniReader
+     */
+    protected $iniReader;
+
+    /**
+     * Constructor
+     */
+    public function __construct()
+    {
+        // Use ASCII 0 as a nest separator; otherwise some of the unusual key names
+        // we have (i.e. in WorldCat.ini search options) will get parsed in
+        // unexpected ways.
+        $this->iniReader = new IniReader();
+        $this->iniReader->setNestSeparator(chr(0));
```

Comment:
I see that this is copied and pasted from VuFind\Config\PluginFactory. I wonder if there's a way we can centralize this logic so there's a single factory for the IniReader instances used to load VuFind configurations. It's entirely possible that's more work than it's worth, but I wanted to at least raise the idea.

Reply:
Ahh, of course, great idea.